client/src/components/Verify.js

```diff
@@ -43,6 +55,14 @@ class Verify extends Component {
         : registeredMethods[0]
       );
     }
+
+    // "All methods" is the combination of registered and available methods
+    if (onSetAllMethods) {
+      this.props.onSetAllMethods([
```

Comment:
Would it make more sense to update the store when the data is initially retrieved, rather than during mount?

Reply:
I think so